src/query_plan.c

```diff
@@ -138,7 +161,8 @@ void Query_OnReopen(RedisModuleKey *k, void *privdata) {
                            (now.tv_nsec - q->execCtx.startTime.tv_nsec);
     // printf("Elapsed: %zdms\n", durationNS / 1000000);
     // Abort on timeout
-    if (durationNS > RSGlobalConfig.queryTimeoutMS * 1000000) {
+    if (durationNS > q->opts.timeoutMS * 1000000) {
+      printf("Setting timed out state!\n");
```

Comment:
remove printf - convert to log if we have a context here

Reply:
Just a leftover